dask_expr/_shuffle.py

```diff
@@ -1034,6 +1034,10 @@ def _lower(self):
         )
         return SortIndexBlockwise(index_set)
 
+    @property
+    def npartitions(self):
```

Comment:
You don't need this, the parent ``npartitions`` implementation does the same thing